brushfire-scalding/src/main/scala/com/stripe/brushfire/scalding/Example.scala

```diff
@@ -2,13 +2,16 @@ package com.stripe.brushfire.scalding
 
 import com.stripe.brushfire._
 import com.twitter.scalding._
+import com.twitter.algebird.AveragedValue
 
-class IrisJob(args: Args) extends TrainerJob(args) {
+abstract class csvJob(args: Args) extends TrainerJob(args) {
   import JsonInjections._
+  val cols: List[String]
+  implicit val stopper = FrequencyStopper[String](10, 3)
+  val error = BrierScoreError[String, Long]
+  implicit val errorOrdering = Ordering.by[AveragedValue, Double] { av => av.value } // Allows us to compare BrierScoreErrors.
 
-  val cols = List("petal-width", "petal-length", "sepal-width", "sepal-length")
-
-  val trainingData =
+  def load_data(): TypedPipe[Instance[String, Double, Map[String, Long]]] = {
```

Comment:
why is this not just `trainingData`, since that calls this and does nothing else?